mod_wsbridge/mod_wsbridge.c

```diff
@@ -873,6 +869,10 @@ static switch_status_t channel_on_init(switch_core_session_t *session)
 		}
 	}
 
+	switch_mutex_lock(globals.mutex);
```

Comment:
If only this is the fix, it looks to me something else is fishy in the module.
Probably worth creating a Tech-debt ticket for track afterwards.